CDP4DiagramEditor/Views/DiagramEditor.xaml

```diff
@@ -118,6 +123,13 @@
                                         SaveCommand="{Binding SaveDiagramCommand}"
                                         dxb:MergingProperties.ElementMergingBehavior="All"
                                         ShowPageBreaks="True" IsManipulationEnabled="True" AllowRotateItems="False" AllowResizeItems="False" AllowApplyAutomaticLayoutForSubordinates="True" AllowApplyAutomaticLayout="True">
+                <diagram:Cdp4DiagramControl.Resources>
+                    <Style x:Key="BaseStyle"
```

Comment:
These styles should no longer be required as the implicit styles in the shell view have now been removed.